spec/models/health_care_application_spec.rb

```diff
@@ -419,15 +422,15 @@
           expect do
             described_class.new(form: { mothersMaidenName: 'm' }.to_json).process!
           end.to raise_error(Common::Exceptions::ValidationErrors)
-        end.to trigger_statsd_increment('api.1010ez.validation_error_short_form')
+        end.to trigger_statsd_increment("#{statsd_key_prefix}.validation_error_short_form")
```

Comment:
Replaced these with a variable to DRY it up a bit. It's a little different than others since the prefix comes from the service and not the class under test, but I still like it better than hardcoding it.